### DISCUSSION TITLE
Tutorial .ipynb do not work

### DISCUSSION THREAD

**Discussion by chvojja22** [Nixtla/statsforecast#827] [Q&A]
Hi, I think the .ipynb script for anomaly detection does not work properly. It throws error for 'levels' not provided. If you provide it, it crashes elswhere, I think it is connected with _filter_series( function where it got filtered too much, incorrectly and the anomalies are not found since it compares it with NaNs

**Comment by Avidar123** [comment]
Did you manage to get it to work by any chance ?

**Comment by jmoralez** [comment]
Hey. We've updated [the tutorial](https://nixtlaverse.nixtla.io/statsforecast/docs/tutorials/anomalydetection.html#recover-insample-forecasts-and-identify-anomalies). Please reopen this if you run into any more issues.
